filter/metadata_lastmodified_test.go

```diff
@@ -9,22 +9,32 @@ import (
 )
 
 func TestMetadataLastModified(t *testing.T) {
+	ptr := func(t time.Time) *time.Time {
+		return &t
+	}
+
 	tests := []struct {
 		name         string
-		lastmodified time.Time
+		lastmodified *time.Time
 		dst          string
 		want         string
 		wantErr      bool
 	}{
 		{
 			name:         "time set",
-			lastmodified: time.Date(2009, time.November, 10, 23, 0, 0, 0, time.UTC),
+			lastmodified: ptr(time.Date(2009, time.November, 10, 23, 0, 0, 0, time.UTC)),
 			dst:          "f2",
 			want:         "1257894000",
 		},
 		{
-			name:         "time empty",
-			lastmodified: time.Time{},
+			name:         "time set at zero",
```

Comment:
nit: better to cal this `zero-value` as it's Go name for a default-initialized value, moreover it's not set to zero explicitely